test/Npgsql.Specification.Tests/NpgsqlDbFactoryFixture.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Data.Common;
+using AdoNet.Specification.Tests;
+
+namespace Npgsql.Specification.Tests
+{
+    public class NpgsqlDbFactoryFixture : IDbFactoryFixture
+    {
+        public DbProviderFactory Factory => NpgsqlFactory.Instance;
+
+        const string DefaultConnectionString =
+            "Server=localhost;User ID=npgsql_tests;Password=npgsql_tests;Database=npgsql_tests;Timeout=0;Command Timeout=0";
+
+        public string ConnectionString =>
+            Environment.GetEnvironmentVariable("NPGSQL_TEST_DB") ?? DefaultConnectionString;
```

Comment:
We should check that the EF Core tests respect `NPGSQL_TEST_DB` too.

Reply:
The Npgsql EF Core tests align with the EF Core project patterns, which is why we have some discrepancies... but it could be useful to have NPGSQL_TEST_DB respected there (or take whatever is used there into the Npgsql ADO project).

Reply:
The EF Core tests already add environment variables to that config, so we might just need to include it in the coalesce:

```c#
public static string DefaultConnection 
    => Config["DefaultConnection"] ?? Config["NPGSQL_TEST_DB"] ?? DefaultConnectionString;
```

Maybe give it a try if you're back in that project today, otherwise I'll play around with it this week.